module/VuFind/src/VuFind/View/Helper/Root/SchemaOrg.php

```diff
@@ -106,4 +111,53 @@ public function getMeta(string $property, string $content, array $attributes = [
     {
         return $this->getTag('meta', compact('property', 'content') + $attributes);
     }
+
+    /**
+     * Get default record types as array.
+     *
+     * @return array
+     */
+    public function getDefaultRecordTypesArray(): array
+    {
+        if ($this->ils->__invoke()->getOfflineMode() !== 'ils-none') {
```

Comment:
For more concise syntax, you can replace `$this->ils->__invoke()->...` with `($this->ils)()->...`. Just note the need for parentheses around `$this->ils` to allow it to be evaluated as a function.